### ISSUE TITLE
urls :( 

### ISSUE THREAD

**Issue by koalahamlet** [lostways/symbolcipher#1]
Hey there ;) 
I started using this for a harmonious project and I found that there were some problems with urls.
Have you noticed that it's not picking up ":" correctly or "?" ? I noticed this when someone posted a youtube link and  the cipher text didn't work well.  I added those to the library and things worked out. 

Will this still work with the alt number characters " !@#$%^&*()_+ " ?

As an aside, I moved the script into a friendly user interface. which is in my github page. feel free to make a pull request!

**Comment by lostways** [comment]
I was actually pretty handicapped when I first wrote this script, as I don't have a mac. So I had to look up the symbols in charts that I could find. It will work for any symbol in the conversion table, if there is a symbol that isn't in the table it will just not convert it.

Thanks for adding the new characters, I'll go back and add those to the original module. I'm also toying with the idea of converting it in to a node module so it's easier to integrate into more projects from a single source.
